var/spack/repos/builtin/packages/parallelio/package.py

```diff
@@ -70,6 +70,12 @@ class Parallelio(CMakePackage):
     # Allow argument mismatch in gfortran versions > 10 for mpi library compatibility
     patch("gfortran.patch", when="@:2.5.8 +fortran %gcc@10:")
 
+    @run_after("install")
+    @when("platform=darwin")
```

Comment:
```suggestion
    @run_after("install", when="platform=darwin")
```

Reply:
Thanks, I didn't realize if could be combined like this.